### PR TITLE
wktui-345  Correct display message when domain status is available.

### PR DESCRIPTION
**Pull request by jshum2479**
Correct message text when domain status is available, copy and paste error from other status type,  there is no extra information in this status.

### CODE REVIEW

File: electron/app/locales/en/webui.json [hunk 1]
```diff
@@ -1186,7 +1186,7 @@
   "k8s-domain-status-checker-domain-status-unknown": "Unknown - Cannot determine the domain status now - check back later",
   "k8s-domain-status-checker-domain-status-complete": "Complete - Domain is deployed and all servers are up and running",
   "k8s-domain-status-checker-domain-status-failed": "Failed - Domain failed to deploy. Reason: {{reason}}",
-  "k8s-domain-status-checker-domain-status-available": "Available - Domain is deployed and some servers are up and running but the domain has not reaching the replicas count goal yet. Reason: {{reason}}",
+  "k8s-domain-status-checker-domain-status-available": "Available - Domain is deployed and some servers are up and running but the domain has not reaching the replicas count goal yet, check again later.",
```
Comment (rakillen) [inline electron/app/locales/en/webui.json:1189]: "has not reaching" -> "has not reached"

Comment (jshum2479) [reply electron/app/locales/en/webui.json]: fixed

File: electron/app/locales/en/webui.json [hunk 2]
```diff
@@ -1186,7 +1186,7 @@
   "k8s-domain-status-checker-domain-status-unknown": "Unknown - Cannot determine the domain status now - check back later",
   "k8s-domain-status-checker-domain-status-complete": "Complete - Domain is deployed and all servers are up and running",
   "k8s-domain-status-checker-domain-status-failed": "Failed - Domain failed to deploy. Reason: {{reason}}",
-  "k8s-domain-status-checker-domain-status-available": "Available - Domain is deployed and some servers are up and running but the domain has not reaching the replicas count goal yet. Reason: {{reason}}",
+  "k8s-domain-status-checker-domain-status-available": "Available - Domain is deployed and some servers are up and running but the domain has not reached the replicas count goal yet, check again later.",
```
Comment (robertpatrick) [inline electron/app/locales/en/webui.json:1189]: Hmm...  In most cases, the reason is available and presumably helps the user quickly and easily understand the state.  I don't feel like the solution to handling the transient situations where the reason is empty is to completely eliminate showing the reasons in all situations.

Comment (jshum2479) [reply electron/app/locales/en/webui.json]: changed the code to put out condition reasons

**Review by robertpatrick — APPROVED** [review]

**Pull request merged**